usaspending_api/awards/migrations/0009_sumaryviews_universal.py

```diff
@@ -0,0 +1,285 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.4 on 2017-11-29 15:31
+from __future__ import unicode_literals
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('awards', '0008_merge_20171116_1601'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='SumaryCfdaNumbersView',
```

Comment:
Where exactly are all these views going to be used? I'm assuming the award search page?

Reply:
The Summary views are separate from the pr and where created because this is the first pr that ran make migrations since the Matview Models where added to django (They do not mess up a user running migrate)